core-plugins/docs/KinesisSink-realtimesink.md

```diff
@@ -0,0 +1,48 @@
+# Amazon Kinesis Stream Real-time Sink
+
+
+Description
+-----------
+Real-time sink that outputs to a specified Amazon Kinesis Stream.
+
+Use Case
+--------
+This sink is used when you want to write to a kinesis stream in real-time. For example, you
+may want to read data from a CDAP stream or any realtime source and write it to a kinesis.
+
+
+Properties
+----------
+**name:** The name of the kinesis stream to output to. Must be a valid kinesis stream name. The kinesis stream
+will be created if it does not exist.
+
+**body.field:** Name of the field in the record that contains the data to be written to
```

Comment:
Based upon the code, it looks like that its also allowed to be an int or long or other simple types.